src/Sql/Sql/ChangeLog.md

```diff
@@ -23,6 +23,7 @@
 ## Version 3.5.1
 
 * Fixed `Get-AzSqlDatabaseImportExportStatus` to report the error encountered
+* Added `ZoneRedundant` parameter to `New-AzSqlDatabaseCopy`, `New-AzSqlDatabaseSecondary` and `Restore-AzSqlDatabase` to enable zone redundant copy, geo secondary and PITR support for hyperscale databases
```

Comment:
The new added descriptive item in changelog.md should be at the top of upcoming release.